### PR TITLE
Bugfix/1286 taginput autocomplete dropdown

### LINKED ISSUE #1286: Taginput should open clicking anywhere within container
Latest Buefy, Chrome, Osx
Taginput with autocomplete and open-on-focus

When clicking inside the visual boundaries you would expect the dropdown to appear, however the @focus listener is on the input contained within and not on the `.taginput-container.is-focusable` container. The input has no visual boundary so it simulates buggy behaviour and not useful if the user needs to know there is data they can choose from:

Example 1:

Example 2:

image

image

Related to this is: when someone deletes a tag, the input gains focus, but the dropdown does not appear.

Would be really great if this could be fixed.

### PR DESCRIPTION
**Pull request by wrabit**
Fixes #1286

### CODE REVIEW

File: src/components/autocomplete/Autocomplete.vue [hunk 1]
```diff
@@ -123,6 +123,15 @@ export default {
                     whiteList.push(child)
                 }
             }
+            if (this.$parent.$options.name === 'BTaginput') {
```
Comment (jtommy) [inline src/components/autocomplete/Autocomplete.vue]: Please use `_isTaginput` to check the component

Comment (wrabit) [reply src/components/autocomplete/Autocomplete.vue]: _isTaginput is not available via `this.$parent._isTaginput`

Comment (wrabit) [reply src/components/autocomplete/Autocomplete.vue]: `this.$parent.$data._isTaginput` is ok?

Comment (jtommy) [reply src/components/autocomplete/Autocomplete.vue]: Sure

**Comment by jtommy** [comment]
Thanks

**Pull request merged**